mubench.pipeline/tests/data/stub_detector.py

```diff
@@ -3,4 +3,5 @@
 
 class StubDetector(Detector):
     def __init__(self):
-        super().__init__("-detectors-", "StubDetector", [], cli_version=RunnerInterfaceTestImpl.TEST_VERSION)
+        self._get_release_info = lambda *_: {'tag': 'latest', 'md5': None, 'cli_version': RunnerInterfaceTestImpl.TEST_VERSION}
```

Comment:
The super constructor uses `cli_version` to get the runner interface, so we need to replace it before the call. I agree that this is not very readable though. We can definitely omit `tag` and `md5`, since they default to None anyway. As for `cli_version`, I am not sure what the best solution is here. We should probably discuss this in our meeting later.